paket.dependencies

```diff
@@ -1,9 +1,10 @@
 source https://nuget.org/api/v2
-nuget BenchmarkDotNet
-nuget BenchmarkDotNet.Diagnostics.Windows
+nuget BenchmarkDotNet 
+nuget BenchmarkDotNet.Diagnostics.Windows 
 nuget FParsec
 nuget FSCheck
 nuget FSharp.Data.TypeProviders
+nuget FSharp.Control.Reactive
```

Comment:
We need it for some of the transformations from other data types to observables, as well as the `bind` operation on observable